web/themes/custom/atrium/templates/media/media--jcc-form--full.html.twig

```diff
@@ -6,6 +6,16 @@
 #}
 <article{{ attributes.addClass(classes) }}>
   {{ title_suffix.contextual_links }}
+
+  {% include '@molecules/blocks/hero/hero.twig' with {
+    hero: {
+      title: media.label ~ ' (' ~ media.field_number.value ~ ')',
+      lead: content.field_description ? content.field_description : '',
+      body: link("Download PDF"|t,  media.field_url.uri, { 'class':['usa-button']} ),
```

Comment:
Using Drupal's link function here - See https://www.drupal.org/docs/8/theming/twig/functions-in-twig-templates

Reply:
Also the "t" function should be used on all strings. This function checks for cross-site scripting and also makes the string available for translation.